examples/bsp/app/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+authors = [
+    "Ulf Lilleengen <lulf@redhat.com>",
+    "Bob McWhirter <bmcwhirt@redhat.com>",
+    "Jens Reimann <jreimann@redhat.com>",
+    "Dejan Bosanac <dbosanac@redhat.com>"
+]
+edition = "2018"
+name = "bsp-blinky-app"
+version = "0.1.0"
+description = "BSP Blinky App"
+keywords = ["bsp"]
+
+[workspace]
+
+
+[dependencies]
+defmt = "0.3"
+drogue-device = { path = "../../../device", default-features = false }
+embassy = { version = "0.1.0", default-features = false, features=["defmt", "time", "time-tick-32768hz"] }
```

Comment:
I don't think we can depend on time-tick-32768hz, it will not be the same for different environments.

Reply:
Ah, I have no idea what it really does. Just copy pasta from somewhere else...